go-controller/pkg/ovn/gress_policy.go

```diff
@@ -0,0 +1,369 @@
+package ovn
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	knet "k8s.io/api/networking/v1"
+	"k8s.io/klog"
+)
+
+type gressPolicy struct {
+	policyNamespace string
+	policyName      string
+	policyType      knet.PolicyType
+	idx             int
+
+	// peerAddressSets points to all the addressSets that hold
+	// the peer pod's IP addresses. We will have one addressSet for
+	// local pods and multiple addressSets that each represent a
+	// peer namespace
+	peerAddressSets map[string]bool
```

Comment:
empty `struct{}` as value instead of `bool`?

Reply:
@abhat was a C&P of the existing code, no change at this time.